hkmc2/shared/src/main/scala/hkmc2/semantics/Term.scala

```diff
@@ -153,7 +199,7 @@ sealed trait Statement extends AutoLocated with ProductWithExtraInfo:
     case Rcd(stats) => stats.flatMap(_.subTerms)
     case Quoted(term) => term :: Nil
     case Unquoted(term) => term :: Nil
-    case New(_, args, rft) => args ::: rft.toList.flatMap(_._2.blk.subTerms)
+    case New(cls, args, rft) => cls :: args ::: rft.toList.flatMap(_._2.blk.subTerms)
```

Comment:
Is it correct to include `cls` to the subStatements?

Reply:
Yes, sure.